main.go

```diff
@@ -159,9 +159,13 @@ var serverHelp = `
     provide a certificate notification email by setting CHISEL_LE_EMAIL.
 
     --tls-ca, a path to a PEM encoded CA certificate bundle or a directory
-    holding multiple PEM encode CA certificate bundle files, which is used to 
-    validate client connections. The provided CA certificates will be used 
-    instead of the system roots. This is commonly used to implement mutual-TLS. 
+    holding multiple PEM encode CA certificate bundle files, which is used to
+    validate client connections. The provided CA certificates will be used
+    instead of the system roots. This is commonly used to implement mutual-TLS.
+
+    --ldap-config, a path to a file containing the ldap authentication
+		configuration.
```

Comment:
it's not super clear what `--ldap-config` does once set. maybe:

```
    --ldap-config, a path to a JSON configuration file, which defines settings used to
    connect to a remote LDAP server for authenticating users. once configured, user
    passwords will be validated against the configured LDAP server. here is an example
    JSON file: { "something": "something" }
```

Reply:
changed as proposed